finagle-http/src/main/scala/com/twitter/finagle/http/Status.scala

```diff
@@ -29,7 +29,47 @@ case class Status(code: Int) {
     }
 }
 
+
+
+
+
 object Status {
+
+  @inline
+  private[finagle] def inRange(lower: Int, upper: Int)
+                              (status: Status): Option[Status] =
+    Some(status).filter(s => s.code >= lower && s.code < upper)
```

Comment:
On the off chance somebody passes in a null Status it would be nice if this were ```Option(status)``` here rather than ```Some(status)```. Same comment applies to line 45.